Tests/kaas/k8s-version-policy/k8s_version_policy.py

```diff
@@ -411,11 +405,11 @@ def check_k8s_version_recency(my_version: K8sVersion, releases_data: list[dict],
         if release.age > PATCH_VERSION_CADENCE:
             # whoops, the cluster should have been updated to this (or a higher version) already!
             return False
-        if my_version in cve_version_list and release.age > CVE_VERSION_CADENCE:
-            # -- three FIXMEs:
-            # (a) can the `in` ever become true, when we have a version vs a set of ranges
-            # (b) if the release still has the CVE, then there is no use if we updated to it?
-            # (c) the standard says "time period MUST be even shorter ... it is RECOMMENDED that ...",
+        is_affected = any(my_version in _range for _range in cve_affected_ranges)
+        if is_affected and release.age > CVE_VERSION_CADENCE:
```

Comment:
Now this is just a matter of taste, but I usually prefer this:
```suggestion
        ranges = [my_version in _range for _range in cve_affected_ranges]
        if ranges and release.age > CVE_VERSION_CADENCE:
```
My reasoning is that this has similar computing cost without throwing away the information.
The information is usually nice to have for debug logging, among other things.
But still, it is a tiny matter of taste.

Reply:
With this suggestion, `ranges` would be a list of `bool`s. And the lhs of the `if` already evaluates to `True` if the list is not empty, even if `my_version` is not affected.

Reply:
By "throwing away information" you mean that I used a generator instead of a list?

Reply:
sorry, I was too quick here, what I meant was rather
```suggestion
        ranges = [_range for _range in cve_affected_ranges if my_version in _range]
        if ranges and release.age > CVE_VERSION_CADENCE:
```

Reply:
By throwing away I meant that we don't have the "witnesses" (or "culprits", if you will). Whatever you would show to someone who wanted to debug this thing.

Reply:
I agree!